deployment/macos/encrypt-keychain.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
When the script is called? I didn't find its mentioning among commited files.

Reply:
This isn't called anywhere, it's only meant to be used manually client-side on a trusted host where the original GPG-encryption of the keychain file is performed.

Reply:
Thanks for clarification :)